xml/planning-architecture-example_configurations.xml

```diff
@@ -68,6 +68,24 @@
     </para>
    </listitem>
   </itemizedlist>
+
+  <para>
+   &o_iron; multi-tenancy feature uses &o_netw; to manage the tenant
+   networks. The interaction between &o_netw; and the physical switch is
+   facilitated by &o_netw;'s Modular Layer 2 (ML2) plugin. The &o_netw; ML2
+   plugin supports drivers to interact with various networks, as each vendor
+   may have their own extensions. Those drivers are referred to as &o_netw; ML2
+   mechanism drivers, or simply mechanism drivers.
+  </para>
+  <para>
+   The &o_iron; multi-tenancy feature has been validated using &ostack;
+   genericswitch mechanism driver. However, if the given physical switch
+   require a different mechanism driver, you must update the input model
+   accordingly. To update the input model with a custom ML2 mechanism driver,
+   add specify the relevant information in the
```

Comment:
either *add* or *specify*. decide!
also, any information on what exactly is "the relevant information"? e.g. some kind of example?

Reply:
Can we let the readers to decide which one they like best: add or specify?

The bug reporter probably assumes that if you know what multi-tenancy is, it's highly likely that you also know what "relevant info" is. So I'm inclined to leave it as it is.